src/js/actions/transform.js

```diff
@@ -1080,28 +1080,27 @@ define(function (require, exports) {
 
     var beforeStartup = function () {
         _artboardTransformHandler = synchronization.debounce(function () {
-            this.dispatch(events.ui.TOGGLE_OVERLAYS, { enabled: true });
-            
             // After each action we call, document model changes
             // so we re-get it
             var appStore = this.flux.store("application"),
                 nextDoc = appStore.getCurrentDocument();
 
-            return this.flux.actions.layers.getLayerOrder(nextDoc, true)
+            return this.flux.actions.layers.resetBounds(nextDoc, nextDoc.layers.allSelected)
```

Comment:
Actually, after testing a bit, I realized that this is correct. We want to resetBounds on the selected layer that was just transformed, THEN reset our selection to make sure any side effects like selection and order change is accounted for.